code/modules/mob/mob.dm

```diff
@@ -60,23 +60,23 @@
 		hud_list[hud] = image('icons/mob/hud.dmi', src, "")
 
 /mob/proc/show_message(msg, type, alt, alt_type)//Message, type of message (1 or 2), alternative message, alt message type (1 or 2)
+	if(!client)	
```

Comment:
please avoid making changes like this to procs you arent altering when this is the only change in the file

Reply:
Fuck how did that get there my bad